build-utils/webpack.prod.js

```diff
@@ -1,31 +1,30 @@
-const commonPaths = require('./common-paths')
-const webpack = require('webpack')
-const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin')
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
```

Comment:
Is this one used?

Reply:
yep, it's quite useful when analysing build size, but it's commented out below